Automattic/newspack-plugin
# feat: integrate fluid to ad unit size control

### All Submissions:






### Changes proposed in this Pull Request:

The current UI for editing ad unit sizes separates the fluid selection from the dropdown of sizes. Although it's handled differently on the API, it shares the same effect on the ad unit, which is registering a new size to allow creatives to display.

This PR changes the `AdUnitSizeControl` to accommodate having `fluid` as a selectable option and removes the checkbox.




### How to test the changes in this Pull Request:

1. Check out this PR and edit an Ad Unit on the Ads wizard
2. Make sure you can perform the following edits smoothly:
    1. Create a new size, select it as fluid
    2. Change the size to a fixed one
    3. Create another size and select it as fluid
    4. Remove the fluid size
    5. Create yet another size, select it as fluid, confirm you can change again to Custom and modify the values
3. Confirm that values already selected are not available on the dropdown options
4. Save an ad unit with multiple sizes (including fluid), reload, and confirm the sizes are as expected

### Other information:
